contingency/contingency-api/src/main/java/com/powsybl/contingency/ContingencyBuilder.java

```diff
@@ -24,12 +24,17 @@ public class ContingencyBuilder {
 
     private final List<ContingencyElement> elements;
 
+    private String name;
+
     ContingencyBuilder(String id) {
         this.id = Objects.requireNonNull(id);
         this.elements = new ArrayList<>();
     }
 
     public Contingency build() {
+        if (name != null) {
```

Comment:
I understand that now you can simplify the code to call just once the constructor with a null name as you has allowed it. Just to be sure: in open-rao, we always check that the name is not null before any use?

Reply:
I will remove that. No previous rao core was "fine" with null name input, but get name was never null because it was using id in case of null name. Now the name can be optional but when there is a get name, a get Id is done if empty (to have same behavior).